### PR TITLE
Fix server error on unknown IBAN

### PR DESCRIPTION
**Pull request by atamanroman**
# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Automatic tests for new functionality are created
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

**Comment by sw1ld** [comment]
Looks good for payment initiation.
Consent creation however contains an iban validation and will return a 400 with the text "Format of certain request fields are not matching the XS2A requirements." if a valid but unkown iban gets passed.

**Comment by atamanroman** [comment]
IMO this is as good as it gets right now - @sw1ld?

File: service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AisConsentSpiImpl.java [hunk 1]
```diff
@@ -41,14 +41,15 @@ public AisConsentSpiImpl(AuthorisationService authorisationService,
       SpiAccountConsent spiAccountConsent,
       AspspConsentData aspspConsentData) {
 
+    // TODO potential bug - we need to check ALL IBANs here? (rat)
```
Comment (sw1ld) [inline service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AisConsentSpiImpl.java:44]: It should be enough to just take the first iban to get a PSU. If this or any other provided Iban does not match the returned PSU, we will receive errors during SCA.

Comment (atamanroman) [reply service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AisConsentSpiImpl.java:44]: Depending on the order in the accounts you might get FORMAT_ERROR or an error during SCA, right? This feels odd. But good enough for now.

**Pull request merged**